test/parallel/test-cluster-dgram-1.js

```diff
@@ -111,5 +111,5 @@ function worker() {
     }
   }, PACKETS_PER_WORKER));
 
-  socket.bind(common.PORT);
+  socket.bind(0);
```

Comment:
@Trott I *think* they'll be using the same port because `exclusive` defaults to `false` and the fd will be shared among workers